src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/ProjectSystem/DefaultProjectSnapshotManagerTest.cs

```diff
@@ -85,16 +83,8 @@ public void Initialize_DoneInCorrectOrderBasedOnInitializePriorityPriority()
             // Building this list in the wrong order so we can verify priority matters
             var triggers = new[] { defaultPriorityTrigger, highPriorityTrigger };
 
-            // This particular test needs to use an actual non-test dispatcher due to threading
-            var dispatcher = new DefaultProjectSnapshotManagerDispatcher();
-
             // Act
-            var projectManager = new TestProjectSnapshotManager(dispatcher, triggers, Workspace)
-            {
-                NotifyTriggersFinishedInitializing = new ManualResetEventSlim()
```

Comment:
This was a hack I added that I don't think we need anymore with this new approach